dockstore-webservice/src/test/java/io/dockstore/webservice/languages/CWLHandlerTest.java

```diff
@@ -57,6 +58,39 @@ public void testDeterminingImageRegistry() {
         Assert.assertEquals("Should be Quay", Registry.QUAY_IO, cwlHandler.determineImageRegistry("quay.io/ucsc_cgl/verifybamid:1.30.0").get());
     }
 
+    @Test
+    public void testDeterminingImageSpecifier() {
+        LanguageHandlerInterface handler = Mockito.mock(LanguageHandlerInterface.class, Mockito.CALLS_REAL_METHODS);
+        Assert.assertEquals(
+                "Should be NO_TAG",
```

Comment:
The comments (first parameters) are redundent. By default, it'll say "expecting NO_TAG, actual: ...." . If I choose to add a comment, I try to explain why.